packages/compat/src/compat-app-builder.ts

```diff
@@ -560,8 +581,7 @@ export class CompatAppBuilder {
     if (!asset) {
       let implicitStyles = this.impliedAssets('implicit-styles', application);
       if (implicitStyles.length > 0) {
-        // we reverse because we want the synthetic vendor style at the top
-        asset = new ConcatenatedAsset('assets/vendor.css', implicitStyles.reverse(), this.resolvableExtensionsPattern);
+        asset = new ConcatenatedAsset('assets/vendor.css', implicitStyles, this.resolvableExtensionsPattern);
```

Comment:
we need this reverse back, but I can impact just the imported stuff somewhere else